### PR TITLE
Starts with adding testing capabilities

### PR DESCRIPTION
**Pull request by pksjce**
With ref to #8, I'm adding testing capability to facilitate future development.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "babel-preset-es2015": "6.16.0",
     "babel-preset-stage-2": "6.17.0",
     "eslint": "3.8.0",
+    "jest": "^17.0.3",
```
Comment (agirton) [inline package.json]: Nit: I like to keep my dependencies exact to help reduce install issues. Can you update so this is exact?

File: src/handleAnswers.js [hunk 2]
```diff
@@ -35,14 +35,16 @@ const createPublicProp = (x) => (x.publicPathRequired && {
...
 
 // loadersMap :: Object -> String -> Object
-const loadersMap = (config) => (x) => (config[x.trim()])
+const loadersMap = (config) => (x = '') => (config[x.trim()])
 
 // createLoadersConfig :: Object -> Object
 // extend answers object with mapped loader configs
-const createLoadersConfig = (x) => ({
-  ...x,
+const createLoadersConfig = ({initialEntry ="", loaders = [], ...rest}) => ({
```
Comment (agirton) [inline src/handleAnswers.js]: Nit: I prefer to use single quotes for consistency and it looks like you used default quotes above. Can you update this to be single quotes?

**Review by agirton — CHANGES_REQUESTED** [review]
This is looking good! Thank you so much for starting this. Can you also add the jest commands to the NPM scripts to help make it easier when deploying a new build or testing updates?

**Comment by pksjce** [comment]
Thanks a ton for taking time out to review! Yeah. Have some time this month, so will try to focus on this :)

**Comment by agirton** [comment]
Thanks again!

**Pull request merged**